app.js

```diff
@@ -0,0 +1,14 @@
+function parseFile(filename){
+    d3.csv(filename, function (data) {
+        return data;
```

Comment:
This function stuff isn't necessary, since we're not doing anything with the csv loading. Cut it down to just d3.csv(filename) to keep it simple.

Reply:
- [ ] Remove unnecessary function(data)